css-color-3/t422-rgba-clip-outside-device-gamut-b-ref.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Reference</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<style type="text/css">
+  body { background: white; color: black; }
+  table { border-spacing: 0 2px; padding: 0; border: none; }
+  td { border: none; padding: 0; height: 1.2em; }
+</style>
+<body>
+  <p><strong>WARNING: This test assumes that the device gamut is sRGB
+  (as it will be for many CRT monitors).</strong></p>
+  <p>Every row in this table should have both columns the same color:</p>
+  <table>
+    <tr>
+      <th style="background:white; color: black">Column 1</th>
+      <th style="background:black; color: white">Column 2</th>
+    </tr>
+    <tr>
+      <td style="background: rgb(102, 255, 102)">&nbsp;</td>
```

Comment:
Sounds like the reference does nothing valuable but confirming that 'background: rgba(-30, 500, -1, 0.6)' renders as 'background: rgb(102, 255, 102)' which the test file addressed.

See http://lists.w3.org/Archives/Public/public-css-testsuite/2015Jan/0027.html